src/components/chart/state-history-chart-line.ts

```diff
@@ -328,23 +328,94 @@ class StateHistoryChartLine extends LitElement {
           }
         });
       } else if (domain === "humidifier") {
+        const hasAction = states.states.some(
+          (entityState) => entityState.attributes?.action
+        );
+        const hasCurrent = states.states.some(
+          (entityState) => entityState.attributes?.current_humidity
+        );
+
+        const hasHumidifying =
```

Comment:
```suggestion
        const isHumidifying =
```

Reply:
Why? I tried to keep it close to the `climate` code. `hasHumidifying` means it *has* action='humidifying' somewhere in the history. Did I get it wrong?

Reply:
climate is using `isHeating` or `isCooling`